src/middlewared/middlewared/plugins/pool_/import_pool.py

```diff
@@ -118,7 +118,7 @@ async def import_pool(self, job, data):
         else:
             pool_name = new_name
 
-        # Let's umount any datasets if root dataset of the new pool is locked and it has unencrypted datasets
```

Comment:
Let's is the proper contraction. If you don't like the apostrophe, then use "let us"